.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules
+/build
```

Comment:
New line in all text files, please. Just configure your IDE to add it automatically